### ISSUE TITLE
Browsable API can't resolve URLs for HyperlinkedRelatedFields when using versioning

### ISSUE THREAD

**Issue by brandoncazander** [tomchristie/django-rest-framework#2477]
Very similar to #2455.

When the Browsable API clones requests to check the permissions on other methods, it doesn't clone the `versioning_scheme` attribute and this causes reverse lookups to fail.

```
ImproperlyConfigured: Could not resolve URL for hyperlinked relationship using view name "aggregator-detail". You may have failed to include the related model in your API, or incorrectly configured the `lookup_field` attribute on this field.
```

**Comment by jpadilla** [comment]
Closed by #2478. Thanks again @brandoncazander!

**Comment by brandoncazander** [comment]
No problem, thanks for the fast action!
